frontend/src/Judge/components/ShowCard.js

```diff
@@ -18,39 +19,93 @@ const ButtonContainer = styled.div`
   bottom: 0;
 `
 
-const ShowCard = props => (
-  <Card>
-    <h2>
-      <Link to={`show/${props.id}`}>{props.name}</Link>
-    </h2>
+function renderCardContents (props) {
+  const now = moment()
+  const judgingStart = moment(props.judgingStart)
+  if (now.isBefore(judgingStart)) {
+    return renderBeforeJudging(props)
+  } else {
+    return renderDuringJudging(props)
+  }
+}
+function renderDuringJudging (props) {
+  return (
+    <div>
+      <Col>
+        <dl>
+          <dt>Judging Progress:</dt>
+          <dd>
+            {props.ownVotes.length} / {props.entries.length}
+          </dd>
+          <dt>Judging Ends:</dt>
+          <dd>
+            <Moment format='YYYY/MM/DD'>{props.judgingEnd}</Moment>
+          </dd>
+        </dl>
+      </Col>
+      <Col>
+        <ButtonContainer>
+          <Button
+            size='lg'
+            style={{ cursor: 'pointer' }}
+            tag={Link}
+            to={`show/${props.id}/vote`}
+            // TODO: Conditionally change the text
+          >
+            {props.ownVotes.length === 0 && props.entries.length === 0
+              ? 'Start'
+              : props.ownVotes.length === props.entries.length
+                ? 'Review'
+                : 'Resume'}
+          </Button>
+        </ButtonContainer>
+      </Col>
+    </div>
+  )
+}
+
+function renderBeforeJudging (props) {
+  return (
     <Col>
+      <div> Judging has not started yet. Come back to vote soon! </div>
       <dl>
+        <dt>Judging Starts:</dt>
+        <dd>
+          <Moment format='YYYY/MM/DD'>{props.judgingStart}</Moment>
+        </dd>
         <dt>Judging Ends:</dt>
         <dd>
           <Moment format='YYYY/MM/DD'>{props.judgingEnd}</Moment>
         </dd>
       </dl>
     </Col>
-    <Col>
-      <ButtonContainer>
-        <Button
-          size='lg'
-          style={{ cursor: 'pointer' }}
-          tag={Link}
-          to={`show/${props.id}/vote`}
-          // TODO: Conditionally change the text
-        >
-          Start
-        </Button>
-      </ButtonContainer>
-    </Col>
+  )
+}
+
+const ShowCard = props => (
+  <Card>
+    <h2>
+      <Link to={`show/${props.id}`}>{props.name}</Link>
+    </h2>
+    {renderCardContents(props)}
```

Comment:
`<CardContents {...props} />`

Reply:
Since we're changing the above functions to stateless components, I would just change this line to be:

```jsx
{moment().isBefore(moment(props.judgingStart))
  ? <BeforeJudging {...props} />
  : <DuringJudging {...props} />}